### DISCUSSION TITLE
Export PDF from mkdocs-materials

### DISCUSSION THREAD

**Discussion by gh-repo-status-check** [squidfunk/mkdocs-material#7795] [Q&A] [answered]
I would like to be able to download my documentation as a PDF. Is there a way to do this?

**Comment by niclasheinz** [comment, marked as answer]
Hi @gh-repo-status-check 

There are many different plugins to create pdfs. I use [mkdocs-exporter](https://github.com/adrienbrignon/mkdocs-exporter).

There are many other plugins for [mkdocs ](https://mkdocs.org)that you can try. Here is a [list ](https://github.com/mkdocs/catalog?tab=readme-ov-file#-site-conversion-pdfepubetc) of registered pdf-export plugins. In this list you will find many more plugins that you can install.

Please note, these plugins are for [mkdocs ](https://mkdocs.org)and mkdocs-materials is only a theme. This means that some plugins might not be compatible with mkdocs-materials

**Reply by gh-repo-status-check** [reply]
Thanks @niclasheinz for your help. I will implement mkdocs-exporter into my doc.
